proto/asn1-pdu/mutated_x509_chain.proto

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+// This protobuf represents an X.509 certificate chain with a set of operations
+// to mutate the chain to create structural relationships between the
+// certificates.
+
+syntax = "proto2";
+
+import "x509_certificate.proto";
+
+package mutated_x509_chain;
```

Comment:
On Friday, we talked about having this be part of the same `x509_certificate` package (recall, multiple protos can be part of the same package), and about removing the redundant "X509"

that is

```
package x509_certificate

message MutatedChain {
```

In the `x509_certificate` package we can talk a bit about `Certificate` and `Chain` as their context (`X.509`) is implied by the package.

Reply:
Great points here -- thank you for helping me simplify things.